jupyterlab/labbox_ephys_widgets_jp/src/extensions/averagewaveforms/AverageWaveformsNew.tsx

```diff
@@ -20,11 +21,11 @@ const AverageWaveformsNew: React.FunctionComponent<SortingViewProps> = (props) =
     //     }
     // }, [hither, setResult, calcMode, setCalcMode])
     // return <div>AAAA: {calcMode} {result + ''}</div>
+    const selectedUnitIdsLookup: {[key: string]: boolean} = selection.selectedUnitIds.reduce((m, uid) => {m[uid + ''] = true; return m}, {} as {[key: string]: boolean})
```

Comment:
The old system was {[key: string]: boolean} which is fast and convenient for lookups, but not the best for storing a set of unit ids. The new system is selection.selectedUnitIds as a number[]. But here we need the lookup, so this code translates from new to old. Wondering if there is something more elegant than this .reduce() call.